### PR TITLE
Adds red button toy to arcade machines

### PR DESCRIPTION
**Pull request by TheVekter**
Added a new red button toy that makes an explosion sound and shakes the screen. Basically it's a play toy version of the Big Red Button that spawns in the syndie base.

Thanks to @Razharas and @TZK13 for helping!

### CODE REVIEW

**Comment by TheVekter** [comment]
It's 8 AM and this is the third time I've remade this PR, holy shit don't let me break it again

File: code/game/objects/items/toys.dm [hunk 1]
```diff
@@ -1091,4 +1091,29 @@ obj/item/toy/cards/deck/syndicate
...
+	icon = 'icons/obj/assemblies.dmi'
+	icon_state = "bigred"
+	w_class = 2.0
+	var/cooldown = 0
+
+/obj/item/toy/redbutton/attack_self(mob/user)
+	if (cooldown < world.time)
+		cooldown = (world.time + 300) // Sets cooldown at 30 seconds
+		user.visible_message("<span class='warning'>[user] presses the big red button.</span>", "<span class='notice'>You press the button, it plays a loud noise!</span>", "<span class='notice'>The button clicks loudly.</span>")
+		//spawn(5) //gia said so
```
Comment (Razharas) [inline code/game/objects/items/toys.dm]: Commented out?

Comment (TheVekter) [inline code/game/objects/items/toys.dm]: Yeah, I thought I deleted it. I'm tired. Lemme try again.

**Comment by TheVekter** [comment]
Helps me keep all the code straight in my head. Plus this way if some other newbie comes behind me they can see what all this shit does without having to ask a million questions (like I did). I can clean up the comments if you want but I don't think it'll matter in the long run.

**Comment by Miauw62** [comment]
Eh, it's a thing a lot of newbies do. You'll learn what should be commented
and what is obvious from the code as you code more.
Op 4-feb.-2015 00:24 schreef "TheVekter" <notifications@github.com>:

**Pull request merged**